packages/node/truffle-config.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
   },
   compilers: {
     solc: {
-      version: "../../node_modules/solc",
+      version: "../contracts/node_modules/solc",
```

Comment:
Stick with `../..`

Reply:
In my testing, I found that that the versions of solcjs installed at `monorepo/node_modules` was older than the version of solcjs installed at `monorepo/contracts/node_modules`, hence this change was needed. Didn't dig into why there are 2 versions installed.

Reply:
It's because of Truffle or ethereum-waffle's dependencies not being fixed I think. We ought to use `moduleResolutions` or perhaps add `solc` to the root `package.json`

Reply:
seems to be fixed after rebase